packages/ui/src/components/va-time-picker/VaTimePickerColumn/VaTimePickerColumn.vue

```diff
@@ -5,90 +5,88 @@
     @keydown.down.stop.prevent="makeActiveNext()"
     @keydown.space.stop.prevent="makeActiveNext(5)"
     @keydown.up.stop.prevent="makeActivePrev()"
+    @scroll="onScroll"
     ref="rootElement"
   >
-    <div class="va-time-picker-cell va-time-picker-cell--fake" />
     <div
       v-for="(item, index) in items" :key="item"
-      :class="{
-        'va-time-picker-cell': true,
-        'va-time-picker-cell--active': index === $props.activeItemIndex
-      }"
+      class="va-time-picker-cell"
+      :class="{ 'va-time-picker-cell--active': index === $props.activeItemIndex }"
```

Comment:
The same here.

Reply:
It won't work here, because here class depends on the current selected value.